src/main/webapp/app/shared/components/clone-repo-button/clone-repo-button.component.html

```diff
@@ -14,7 +14,7 @@
         container="body"
     ></button>
     <ng-template #popContent>
-        @if (useSsh && !localVCEnabled) {
+        @if ((useSsh && !user.sshPublicKey) || gitlabVCEnabled) {
```

Comment:
Only show the warning if the user wants to use SSH.
```suggestion
        @if (useSsh && (!user.sshPublicKey || gitlabVCEnabled)) {
```

Reply:
good suggestion :+1: